apps/web/pages/en/haskolanam/index.tsx

```diff
@@ -0,0 +1,14 @@
+import withApollo from '@island.is/web/graphql/withApollo'
+import { withLocale } from '@island.is/web/i18n'
+import UniversitySearch from '@island.is/web/screens/UniversitySearch/UniversitySearch'
+import { getServerSidePropsWrapper } from '@island.is/web/utils/getServerSidePropsWrapper'
+
+const Screen = withApollo(
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore make web strict
+  withLocale('en')(UniversitySearch),
+)
```

Comment:
@rafnarnason 
I don't remember but was this in line with the TS we headed the other day?